doc/api/crypto.md

```diff
@@ -656,6 +656,54 @@ assert.strictEqual(aliceSecret.toString('hex'), bobSecret.toString('hex'));
 // OK
 ```
 
+### ECDH.convertKey(key, curve[, inputEncoding[, outputEncoding[, format]]])
+<!-- YAML
+added: REPLACEME
+-->
+
+- `key` {string | Buffer | TypedArray | DataView}
+- `curve` {string}
+- `inputEncoding` {string}
+- `outputEncoding` {string}
+- `format` {string} Defaults to `uncompressed`.
```

Comment:
Nit:
```diff
-- `format` {string} Defaults to `uncompressed`.
+- `format` {string} **Default:** `uncompressed`
```
As per https://github.com/nodejs/node/pull/19737 (will be backported, but without this fix).

Reply:
Thanks. Updated.